features/cellular/framework/targets/UBLOX/AT/UBLOX_AT.cpp

```diff
@@ -22,8 +22,16 @@
 using namespace mbed;
 using namespace events;
 
+static const AT_CellularBase::SupportedFeature unsupported_features[] =  {
```

Comment:
#ifdef TARGET_UBLOX_C030_R410M

Reply:
Do you want me to add #ifdef over that static const `AT_CellularBase::SupportedFeature unsupported_features` ?

Reply:
To avoid warning: unused variable.

Reply:
I would say yes, otherwise for other targets, this might be in the final image?

Reply:
Updated. Please review now.